pg-index-health-spring-boot-demo/src/main/resources/application.yaml

```diff
@@ -8,8 +8,12 @@ spring:
         user:
             name: demouser
             password: testpwd123
-
-logging:
+    jpa:
+        properties:
+            javax.persistence.query.timeout: 30000
```

Comment:
Please add TODO with a link to https://github.com/spring-projects/spring-boot/issues/33866
We need to rename this property after migration to Spring Boot 3